### PR TITLE
CIV-7857 evidence upload ft types of document

### PR DESCRIPTION
**Pull request by laarmada**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-7857

### Change description ###

the screen for Fast track " What types of documents do you want to upload.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by m-meulendijks-v1 — DISMISSED** [review]

File: src/main/common/models/document/documentType.ts [hunk 1]
```diff
@@ -23,3 +22,32 @@ export enum DocumentUri {
...
+  DOCUMENTS_REFERRED='DOCUMENTS_REFERRED'
+
+}
+export enum EvidenceUploadDisclosure {
+
+  DISCLOSURE_LIST='DISCLOSURE_LIST',
+  DOCUMENTS_FOR_DISCLOSURE='DOCUMENTS_FOR_DISCLOSURE'
+
+}
+export enum EvidenceUploadExpert {
```
Comment (hmcts-version1-ignacio) [inline src/main/common/models/document/documentType.ts:40]: Just a minimum detail, there is no blank line left here, while it is in the others

File: src/test/utils/errorMessageTestConstants.ts [hunk 2]
```diff
@@ -143,4 +143,5 @@ export const enum TestMessages {
   INTEREST_START_DATE = 'Enter the date you want to claim interest from',
   REASON_EXPLANATION = 'Briefly explain your claim',
   CLAIMANT_REJECTION_REASON = 'Enter why you rejected repayment plan',
+  VALID_ENTER_AT_LEAST_ONE_UPLOAD='You must select at least one type of document'
```
Comment (hmcts-version1-ignacio) [inline src/test/utils/errorMessageTestConstants.ts:146]: Just another very minor thing, The spaces between the "=" are missing on this one.

File: src/test/utils/mocks/civilClaimResponseMock.json [hunk 3]
```diff
@@ -278,6 +278,20 @@
...
+    },
+    "caseProgression": {
+      "claimantUploadDocuments":{
+          "trial": [
+            {"documentType": "CASE_SUMMARY", "selected": true},
+            {"documentType": "DOCUMENTARY", "selected": true}
+          ]
+      },
+      "defendantUploadDocuments":{
+        "trial": [
```
Comment (hmcts-version1-ignacio) [inline src/test/utils/mocks/civilClaimResponseMock.json:290]: Another minor thing. Spaces before "trial" are different from the previous one.

**Review by kurt-uk — DISMISSED** [review]

**Review by LeonardoPalmeiro — DISMISSED** [review]

File: src/main/common/models/evidenceUpload/documentEvidenceForTrial.ts
[the remark this answers is not in the document]
Comment (laarmada) [reply src/main/common/models/evidenceUpload/documentEvidenceForTrial.ts:1]: remove it

File: src/main/common/models/document/documentType.ts
[the remark this answers is not in the document]
Comment (laarmada) [reply src/main/common/models/document/documentType.ts:1]: done

**Review by LeonardoPalmeiro — APPROVED** [review]

File: src/test/utils/errorMessageTestConstants.ts [hunk 4]
```diff
@@ -143,4 +143,5 @@ export const enum TestMessages {
   INTEREST_START_DATE = 'Enter the date you want to claim interest from',
   REASON_EXPLANATION = 'Briefly explain your claim',
   CLAIMANT_REJECTION_REASON = 'Enter why you rejected repayment plan',
+  VALID_ENTER_AT_LEAST_ONE_UPLOAD ='You must select at least one type of document'
```
Comment (hmcts-version1-ignacio) [inline src/test/utils/errorMessageTestConstants.ts:146]: Just a very minor thing here, a space after the "=" is missing!

**Review by kurt-uk — APPROVED** [review]

**Review by kurt-uk — DISMISSED** [review]

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by m-meulendijks-v1 — APPROVED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Pull request merged**